web/massastation/src/pages/Store/StoreSection/StorePlugin.tsx

```diff
@@ -1,27 +1,33 @@
-import { useEffect } from 'react';
+import { useEffect, useState } from 'react';
 import { usePost } from '@/custom/api';
 
 import { Certificate, Plugin } from '@massalabs/react-ui-kit';
 import { massalabsNomination } from '@/const';
-import { FiDownload } from 'react-icons/fi';
+import { FiAlertTriangle, FiDownload } from 'react-icons/fi';
 import { PluginCardLoading } from '../PluginCardLoading';
 import { MassaStoreModel } from '@/models';
-
+import Intl from '@/i18n/i18n';
 interface StorePluginProps {
   plugin: MassaStoreModel;
   refetch: () => void;
 }
 
 function StorePlugin(props: StorePluginProps) {
   const { plugin, refetch } = props;
+  const [showTooltip, setShowTooltip] = useState(false);
```

Comment:
```suggestion
  const [showTooltip, setShowTooltip] = useState<boolean>(false);
```

Can you add the type for this state? 🤔

Reply:
it's not implicit with the initial value given?

Reply:
> it's not implicit with the initial value given?

Maybe! But I'm not sure because I believe that at any time the developer can make a:
```typescript
setShowTooltip("hello");
```